test/everypolitician/popolo/json_test.rb

```diff
@@ -15,16 +15,16 @@ def test_legislative_periods_ignores_other_event_types
     assert_equal 2, popolo_json.legislative_periods.count
   end
 
-  def test_current_legislative_period_returns_correct_term
-    assert_equal 'Term 2', popolo_json.current_legislative_period.name
+  def test_latest_legislative_period_returns_correct_term
+    assert_equal 'Term 2', popolo_json.latest_legislative_period.name
   end
 
   def test_that_terms_returns_the_same_as_legislative_periods
     assert_equal popolo_json.terms, popolo_json.legislative_periods
   end
 
-  def test_current_term_returns_the_same_as_current_legislative_period
-    assert_equal popolo_json.current_term, popolo_json.current_legislative_period
+  def test_latest_term_returns_the_same_as_latest_legislative_period
+    assert_equal popolo_json.latest_term, popolo_json.latest_legislative_period
```

Comment:
probably worth leaving at least one test with the `current_` form too, just to make sure it's still working (and to be able to hook into once we start emitting deprecation warnings…)